gradle/libs.versions.toml

```diff
@@ -9,22 +9,22 @@ kronosNTP = "0.0.1-alpha11"
 # Android
 androidDesugaringSdk = "2.0.4"
 androidToolsPlugin = "8.6.1"
-androidXAnnotations = "1.1.0"
-androidXAppCompat = "1.3.0"
-androidXCollection = "1.1.0"
-androidXComposeBom = "2023.10.01"
+androidXAnnotations = "1.9.1"
+androidXAppCompat = "1.4.2" # Next version will bring coroutines
+androidXCollection = "1.4.5"
+androidXComposeBom = "2023.10.01" # Next version will bring coroutines
```

Comment:
having coroutines as a part of Compose is fine I think, it is an essential part of the apps using Compose in most cases, so we can go higher here. The total size of transitive dependencies for `dd-sdk-android-compose` is big already anyway.